### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by rakshith91** [comment]
> Do you want to update https://github.com/Azure/azure-sdk/blob/master/releases/2020-09/python.md as well?

updated

**Review by xiangyan99 — COMMENTED** [review]
Please add Installation Instructions & Changelog as well. :)

File: releases/2020-09/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 ## Release Highlights
 
 * Azure Form Recognizer GA release supporting v2.0 GA version of the Form Recognizer service.
-*
+* Azure EventGrid first preview (v2.0.0b1) is released.
```
Comment (rakshith91) [inline releases/2020-09/index.md:15]:
```suggestion
* Azure Event Grid v2 preview SDK (v2.0.0b1) is released with support for CloudEvent
```

File: releases/2020-09/python.md [hunk 2]
```diff
@@ -18,14 +18,15 @@ The Azure SDK team is pleased to make available the September 2020 client librar
...
-- _Add packages_
+- Azure Event Grid preview 1 (2.0.0b1)
 
 ## Installation Instructions
 
 To install the latest preview version of the packages, copy and paste the following commands into a terminal:
 
 ```bash
 pip install azure-ai-formrecognizer
+pip install azure-eventgrid
```
Comment (xiangyan99) [inline releases/2020-09/python.md:29]: do we need --pre?

Comment (rakshith91) [inline releases/2020-09/python.md:29]:
```suggestion
pip install azure-eventgrid --pre
```

Comment (rakshith91) [reply releases/2020-09/python.md]: totally :)

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**